contrib/src/main/java/com/datatorrent/contrib/avro/AvroToPojo.java

```diff
@@ -88,6 +88,10 @@
   @VisibleForTesting
   int fieldErrorCount = 0;
 
+  public AvroToPojo() {
```

Comment:
In that case it can be private with a comment. You find many occurrences of the same elsewhere.